src/sentry/static/sentry/app/components/events/interfaces/crashContent/index.tsx

```diff
@@ -11,7 +11,7 @@ type Props = Pick<
   'stackType' | 'stackView' | 'projectId' | 'event' | 'newestFirst'
 > & {
   exception?: ExceptionType;
-  stacktrace?: React.ComponentProps<typeof Stacktrace>['stacktrace'];
+  stacktrace?: ExceptionValue['stacktrace'];
```

Comment:
these changes were necessary in this PR. I also intend to clean these types in a follow-up PR